### PR TITLE
Adding Duration ptr helper.

### PR DESCRIPTION
**Pull request by n3wscott**

### CODE REVIEW

**Comment by jonjohnsonjr** [comment]
Can't you take the address of a `time.Time` literal?

**Comment by n3wscott** [comment]
Time removed.

File: ptr/ptr_test.go [hunk 1]
```diff
@@ -49,3 +52,19 @@ func TestString(t *testing.T) {
...
+
+func TestDuration(t *testing.T) {
+	want := 42 * time.Second
+	gotPtr := Duration(want)
+	if want != *gotPtr {
+		t.Errorf("Duration() = &%v, wanted %v", *gotPtr, want)
+	}
+}
+
+func TestDuration_Const(t *testing.T) {
```
Comment (vagababov) [inline ptr/ptr_test.go]:
```suggestion
func TestDurationConst(t *testing.T) {
```

Comment (n3wscott) [reply ptr/ptr_test.go]: nack. 

snake case cases bro.

**Comment by vagababov** [comment]
💣

**Pull request merged**